x-pack/plugins/security/public/management/role_mappings/edit_role_mapping/mapping_info_panel/mapping_info_panel.tsx

```diff
@@ -61,7 +61,7 @@ export class MappingInfoPanel extends Component<Props, State> {
   }
   public render() {
     return (
-      <EuiPanel>
+      <EuiPanel hasShadow={false} hasBorder={true}>
```

Comment:
**nit:** I think `hasShadow` defaults to false when `hasBorder` is enabled so you could simplify this to:

```js
<EuiPanel hasBorder>
```